content/sensu-go/6.4/operations/maintain-sensu/upgrade.md

```diff
@@ -34,12 +34,16 @@ To confirm the installed version, run `sensu-agent version`, `sensu-backend vers
 ## Upgrade to Sensu Go 6.4.0 from any previous version
 
 In Sensu Go 6.4.0, we upgraded the embedded etcd version from 3.3.22 to 3.5.0.
-As a result, 6.4.0 is not backward-compatible with previous Sensu versions.
+As a result, for deployments use embedded etcd, 6.4.0 is not backward-compatible with previous versions of the Sensu backend.
```

Comment:
This looks good but I think the grammar is slightly wrong :sweat_smile:

Reply:
So it is....thank you!